go.mod

```diff
@@ -25,7 +24,8 @@ require (
 	k8s.io/apiextensions-apiserver v0.0.0
 	k8s.io/apimachinery v0.0.0
 	k8s.io/apiserver v0.0.0
-	k8s.io/client-go v0.0.0
+	k8s.io/cli-runtime v0.23.1
+	k8s.io/client-go v0.23.1
```

Comment:
why does go mod add tags here? Not wrong, replaced anyway, just curious.

Reply:
I brought the v0.0.0 back and it works well. Probably VSCode overdoing it.